docs/devel/issue-priorities.md

```diff
@@ -0,0 +1,6 @@
+These are the meanings of the labels priority/P0 ... priority/P3 that we apply to issues in order to try to prioritize them relative to each other.  We try to apply these priority labels consistently across the entire project, but if you notice an issue that you believe to be misprioritized, please do let us know and we will evaluate your counter-proposal.
+
+- **priority/P0**: Must be actively worked on as someone's top priority right now. Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. TL's of teams are responsible for making sure that all P0's in their area are being actively worked on.  Examples include user-visible bugs in core features, broken builds or tests and critical security issues.
+- **priority/P1**: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
```

Comment:
"in time for the next release" - the next point release (1.2, 1.3) or the next every-two-week binary cut?